enterprise/backend/src/metabase_enterprise/transforms_python/settings.clj

```diff
@@ -0,0 +1,117 @@
+(ns metabase-enterprise.transforms-python.settings
+  (:require
+   [metabase.config.core :as config]
+   [metabase.settings.core :as setting]
+   [metabase.util.i18n :refer [deferred-tru]]))
+
+(def ^:private is-not-prod? (or config/is-dev? config/is-test?))
+
+(setting/defsetting python-execution-server-url
+  (deferred-tru "URL for the Python execution server that runs transform functions.")
+  :type :string
+  :visibility :admin
+  :default (when is-not-prod? "http://localhost:5001")
```

Comment:
I've got mixed feeling about configuring all these dev defaults.

Not sure if there's a better way for us to inject them. We could use `config.yml` for ourselves, but maybe that's too inconvenient for future developers.